### ISSUE TITLE
Keyword case convention violated with embedded argument

### ISSUE THREAD

**Issue by UniqueActive** [MarketSquare/robotframework-robocop#1555]
Hi,

I get warnings about the case convention when using a period in an embedded argument for a keyword. This seems to be a bug because it works when I capitalise the letter after the period, like with this minimal example:

(using `wrong-case-in-keyword-name.convention=first_word_capitalized`)

``` robot
*** Variables ***
${TEST_FILE}    test.txt


*** Test Cases ***
Sample test case
    Process file test with keyword          # No warning
    Process file test.txt with keyword      # Does not follow case convention
    Process file test.Txt with keyword      # No warning !!!
    Process file ${TEST_FILE} with keyword  # No warning


*** Keywords ***
Process file ${name} with keyword
    Log    ${name}
```

I am using version 6.10.1

See #1169 for a similar issue.

**Comment by bhirsz** [comment]
Ah you're right, it shouldn't produce a warning - I forgot about the first letter capitalized mode.
